rtptransceiver.go

```diff
@@ -30,6 +30,12 @@ func (t *RTPTransceiver) Sender() *RTPSender {
 	return nil
 }
 
+// SetSender sets the RTPSender and Track to current transceiver
+func (t *RTPTransceiver) SetSender(s *RTPSender, track *Track) error {
```

Comment:
I think you want `ReplaceTrack`. This isn't easily possible with the API we have today :(

With the PR I am landing you should have this with-in the week though!

Reply:
@Sean-Der  We are doing this PR to reuse inactive transceivers, since atm in the SFU, when a track is removed the transceiver is set to inactive, and left there, so after some joins and rejoins we end up with a gigantic sdp full of inactive transceivers. These helper is to allow reuse the inactive sender transceiver.

Reply:
@OrlandoCo oh nice, thanks makes sense to me!

If we ever come back to this I looked through [AddTrack](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-addtrack) and friends and looks like we can't accomplish this with any standard APIs :(

Reply:
@Sean-Der An option should be not setting the sender to `nil` and replace the track.